src/test/java/uk/gov/hmcts/reform/rsecheck/errorhandling/GlobalExceptionHandlerTest.java

```diff
@@ -21,20 +21,9 @@ public void testHandleSubscriptionNotFoundMethod() {
 
         GlobalExceptionHandler globalExceptionHandler = new GlobalExceptionHandler();
 
-        PublicationNotFoundException publicationNotFoundException
```

Comment:
deleting the test is not the way to go about it, we still need to test the publication not found exception